src/utils/img.js

```diff
@@ -18,9 +18,10 @@
  * Sets the img src to the first url in the srcset if srcset is defined but
  * src is not for browsers that do not support srcset.
  * @param {!Element} img
- * @param {string} srcset
+ * @param {!Element} element
  */
-export function guaranteeSrcForSrcsetUnsupportedBrowsers(img, srcset) {
+export function guaranteeSrcForSrcsetUnsupportedBrowsers(img, element) {
+  const srcset = element.getAttribute('srcset');
```

Comment:
In all cases so far, `img.src === element.src` and `img.srcset === element.srcsset`. So, only `img` parameter is necessary.

Reply:
That means that the propagation always needs to happen before this function call...
which is pretty reasonable. Made the change. 

Thank you for pointing that out!